packages/ngtools/webpack/src/angular_compiler_plugin.ts

```diff
@@ -861,16 +845,12 @@ export class AngularCompilerPlugin {
       // We need to run the `listLazyRoutes` the first time because it also navigates libraries
```

Comment:
These comments no longer apply.
And the enclosing `entryModule` conditional shouldn't be needed either.

Reply:
The `entryModule` check is still required as without it a number of lazy loading tests fails. Mainly the once that are using `import` or `System.import` as it will fail with an error

```
Cannot read property 'isComponent' of undefined
```